service/src/test/java/bio/terra/tanagra/service/StudyServiceTest.java

```diff
@@ -82,6 +83,16 @@ void createUpdateDelete() throws InterruptedException {
     // Delete.
     studyService.deleteStudy(createdStudy.getId());
     assertThrows(NotFoundException.class, () -> studyService.getStudy(createdStudy.getId()));
+
+    // Create with id.
+    String id = "aou-rw-26297573";
+    Study studyWithId =
+        studyService.createStudy(
+            Study.builder().id(id).displayName(displayName).description(description),
+            createdByEmail);
+    assertNotNull(studyWithId);
+    LOGGER.info("Created study {} at {}", studyWithId.getId(), studyWithId.getCreated());
+    assertEquals(id, studyWithId.getId());
```

Comment:
RE duplicate ids comment above: Maybe a check here would be helpful to make sure we don't break this in the future? e.g. add below

```
    // Create with duplicate id.
    assertThrows(BadRequestException.class, () -> studyService.createStudy(Study.builder().id(id), createdByEmail));

```